docs/infra/set-up-app-env.md

```diff
@@ -28,12 +28,19 @@ Depending on the value of `has_database` in the [app-config module](/infra/app/a
 
 ## 2. Build and publish the application to the application build repository
 
-Before creating the application resources, you'll need to first build and publish at least one image to the application build repository. Do that by running
+Before creating the application resources, you'll need to first build and publish at least one image to the application build repository. 
 
-```bash
-make release-build
-make release-publish
-```
+There are two ways to do this:
+
+1. Trigger the "Build and Publish" workflow from your repo's GitHub Actions tab. This option requires that the `role-to-assume` GitHub workflow variable has already been setup as part of the overall infra account setup process.
```

Comment:
more of an fyi, but this won't be true after i merge this PR: https://github.com/navapbc/platform-test/pull/16 (really struggling to get a reviewer for that PR)